prog/vm/host_nexus.rb

```diff
@@ -23,7 +23,7 @@ def self.assemble(sshable_hostname, location: "hetzner-hel1", net6: nil, ndp_nee
       Strand.create(
         prog: "Vm::HostNexus",
         label: "start",
-        stack: [{"spdk_version" => spdk_version}]
+        stack: [{"spdk_version" => spdk_version, "download_default_boot_image" => download_default_boot_image}]
```

Comment:
What if we change `Config.default_boot_image_name` to an array, and pass a lis of image names for `download_default_boot_image` instead of a boolean? We could still keep the default list with a single item: "ubuntu-jammy". If the list is empty or nil, the download can skipped. This would also allow operators to download images while setting up hosts. For example, they could pass a list to `Vm::HostNexus.assemble(....., default_boot_images: ["ubuntu-jammy", "github-gpu-ubuntu-2204", "github-ubuntu-2004"])` and no need for manual downloads after setup.